src/backends/pgfplotsx.jl

```diff
@@ -18,14 +18,20 @@ end
 function (pgfx_plot::PGFPlotsXPlot)(plt::Plot{PGFPlotsXBackend})
     if !pgfx_plot.is_created
         cols, rows = size(plt.layout.grid)
-        the_plot = PGFPlotsX.TikzPicture(PGFPlotsX.GroupPlot(
-            PGFPlotsX.Options(
-                "group style" => PGFPlotsX.Options(
-                    "group size" => string(cols)*" by "*string(rows)
+        the_plot = PGFPlotsX.TikzPicture()
+        # the combination of groupplot and polaraxis is broken in pgfplots
```

Comment:
but in PGFPlotsX or in latex pgfplots?

Reply:
latex pgfplots, try to wrap a `PolarAxis` in a `GroupPlot`

Reply:
cf. https://tex.stackexchange.com/questions/112331/latex-pgf-groupplot-polar-axis